client/src/Components/Common/Comment/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Row, Col, Typography } from 'antd';
+import PropTypes from 'prop-types';
+
+import Rating from '../Rating';
+import MainButton from '../MainButton';
+import UserImage from '../UserImage';
+import './style.css';
+
+const { Title, Text } = Typography;
+
+const Comment = ({
+  isAdmin,
+  userName,
+  maxWidth,
+  commentText,
+  date,
+  rateValue,
+  onAdminClick,
+  ...otherProps
+}) => (
+  <div className="Comment" style={{ maxWidth }} {...otherProps}>
```

Comment:
What's the point of `maxWidth`?
There's only 2 cases `isAdmin` and the normal comment